docs/getting-started.md

```diff
@@ -50,13 +60,29 @@ helm repo add antrea https://charts.antrea.io
 helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true
 ```
 
+Ensure `flowExporter.enable` flag is set to true when the version of Antrea is later than v1.13.
```

Comment:
```suggestion
Ensure `flowExporter.enable` flag is set to true when using Antrea v1.13 or greater.
```
I feel "when the version of Antrea is later than v1.13" is not appropriate as it seems excludes v1.13

Reply:
Thanks for reviewing.
We have similar paragraph, but we don't have helm instruction there.
I've updated the document. Please take a look to see if it is clear now?